desktop/apps/editorial_features/components/venice_2017/stylesheets/video.styl

```diff
@@ -20,6 +20,8 @@ i.icon-close.venice-video__close
     min-height 50px
     z-index 12
     float right
+.unpublished #controls
+  display none
```

Comment:
Could you tell me the reason for adding this? I think I removed it at one point because if you land on a preview page, then navigate to a video and play, the controls went missing. I see the video player as always being "visible", and the carousel being responsible for overlays.

Related, I wonder if there's a more React-y way to approach these states, like putting in a `data-state` attribute on the `.venice-feature` root and making CSS decision based on that? Would love to hear your thoughts!